CHANGELOG.adoc

```diff
@@ -67,6 +67,7 @@ More existing APIs will be migrated to this new common pattern.
 - https://github.com/eclipse-sirius/sirius-web/issues/3650[#3650] [diagram] Fix potential NPE in DiagramNavigator and Node toString method 
 - https://github.com/eclipse-sirius/sirius-web/issues/3649[#3649] [sirius-web] Restore support for Related elements view icons
 - https://github.com/eclipse-sirius/sirius-web/issues/3630[#3630] [sirius-web] Restore support for the deletion of dangling representations
+- https://github.com/eclipse-sirius/sirius-web/issues/3666[#3666] [core] Enable EMF's entrinsicIDToEObjectMap for JSON resources to support languages which use modeled identifiers (e.g. SysMLv2).
```

Comment:
entrinsicIDToEObjectMap => intrinsicIDToEObjectMap

Reply:
Done. Also fixed in the commit message.